brainscore_language/models/gpt/__init__.py

```diff
@@ -0,0 +1,12 @@
+from brainscore_language import model_registry
+from brainscore_language import ArtificialSubject
+from brainscore_language.model_helpers.huggingface import HuggingfaceSubject
+
+# layer assignment based on choosing the maximally scoring layer on Pereira2018-encoding from
+# https://github.com/mschrimpf/neural-nlp/blob/master/precomputed-scores.csv
+
+model_registry['distilgpt2'] = lambda: HuggingfaceSubject(model_id='distilgpt2', region_layer_mapping={
+    ArtificialSubject.RecordingTarget.language_system: 'transformer.h.5.mlp.dropout'})
```

Comment:
Cool! I assume that going forward, we would base this off the new benchmarks (and separate into modality)?

Reply:
It sounds more intuitive to me to keep the same layer per region and then change neural activity based on the modality, but it's up to you / future model submitters

Reply:
okay, sorry, my "new benchmarks" was a bit ambiguous: I meant that we should base it off the updated Pereira benchmark and as you mention keep the same layer per region